pkg/remote/aws/repository/ecr_repository_test.go

```diff
@@ -88,3 +88,54 @@ func Test_ecrRepository_ListAllRepositories(t *testing.T) {
 		})
 	}
 }
+
+func Test_ecrRepository_GetRepositoryPolicy(t *testing.T) {
+
+	tests := []struct {
+		name    string
+		mocks   func(client *awstest.MockFakeECR)
+		want    *ecr.GetRepositoryPolicyOutput
+		wantErr error
+	}{
+		{
+			name: "Get repository policy",
+			mocks: func(client *awstest.MockFakeECR) {
+				client.On("GetRepositoryPolicy",
+					mock.Anything,
```

Comment:
You can define a common fake repository above and use it here, instead of `mock.Anything`